src/dsp_tools/xmllib/models/resource.py

```diff
@@ -116,7 +116,7 @@ def serialise(self) -> etree._Element:
         res_ele = self._serialise_resource_element()
         if self.file_value:
             res_ele.append(self.file_value.serialise())
-        res_ele.extend(self._serialise_values())
+        res_ele.extend(serialise_values(self.values))
```

Comment:
Making the serialisation of the resource generic is a next step.
I don't think it is a problem to remove the serialise function in the values, because it was never user-facing. If we want to keep it public for the resource we can decide later.